schemas/dab.draft.schema.json

```diff
@@ -175,6 +175,23 @@
             "enabled": {
               "type": "boolean",
               "description": "Allow enabling/disabling GraphQL requests for all entities."
+            },
+            "nested-mutations": {
+              "type": "object",
+              "description": "Feature flags for nested mutation operations",
+              "properties": {
+                "create":{
+                  "type": "object",
+                  "description": "Options for nested create operations",
```

Comment:
add "additionalProperties": false.  So, that if any property is there or if there is a typo, it is being caught by the schema validator